beacon-chain/state/state-native/getters_misc.go

```diff
@@ -87,3 +87,21 @@ func (b *BeaconState) balancesLength() int {
 
 	return len(b.balances)
 }
+
+// HistoricalSummaries of the beacon state.
+func (b *BeaconState) HistoricalSummaries() []*ethpb.HistoricalSummary {
+	if b.historicalSummaries == nil {
```

Comment:
I skipped the versioning checks here and `HistoricalSummaries` because the list would just be emptied and error out upon return. I don't think it's a big deal, and I didn't feel like changing the return argument to include an error. Happy to change it otherwise

Reply:
Don't most of the other getters return an error when a property is accessed on a version of the state which does not have that property?